src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md

```diff
@@ -3,26 +3,28 @@ title: 'Requirements'
 excerpt: 'Requirements for using xk6-disruptor in your test scripts'
 ---
 
-The xk6-disruptor is a k6 extension. 
-To use it in a k6 test script, you need to bundle a k6 extension that includes the disruptor.
-Refer to the [Installation](/javascript-api/xk6-disruptor/get-started/installation) section  for instructions on how to get this custom build.
+At this moment,
```

Comment:
No at this moment. It is an extension. And it is intended for systems running kubernetes. And it requires kubernetes 1.23. I prefer the original phrasing.